src/sentry/data/samples/transaction-consecutive-db-queries.json

```diff
@@ -0,0 +1,157 @@
+    "mark.fid": {"value": 1675379261.44, "unit": "none"},
+    "mark.fp": {"value": 1675379263.695, "unit": "none"},
+    "mark.lcp": {"value": 1675379264.244, "unit": "none"}
+  },
+  "metadata": {"location": "/consecutive-db/", "title": "/consecutive-db/"},
+  "modules": {"my.package": "1.0.0"},
+  "nodestore_insert": 1675379263.836825,
+  "received": 1675379263.788509,
+  "request": {
+    "url": "http://countries:8010/country_by_code/",
+    "method": "GET",
+    "query_string": [["code", "CAN"]],
+    "headers": [
+      ["Accept", "*/*"],
+      ["Accept-Encoding", "gzip, deflate"],
+      ["Connection", "keep-alive"],
+      ["Content-Length", ""],
+      ["Content-Type", "text/plain"],
+      ["Host", "countries:8010"],
+      ["Referer", "fixtures.transaction"],
+      ["Sentry-Trace", "a7d67cf796774551a95be6543cacd459-8988cec7cc0779c1-1"],
+      ["User-Agent", "python-requests/2.22.0"]
+    ],
+    "env": {"SERVER_NAME": "a90286977562", "SERVER_PORT": "8010"},
+    "inferred_content_type": "text/plain"
+  },
+  "span_grouping_config": {"id": "default:2022-10-27"},
+  "spans": [
+    {
+      "timestamp": 1675379261.736703,
+      "start_timestamp": 1675379261.436703,
+      "description": "new",
+      "op": "django.view",
+      "span_id": "f32e597a998a4e49",
+      "parent_span_id": "4e3384c3c41c4f05",
+      "trace_id": "9a2e3dbcabf2497b8dc79c5b7e4d0383",
+      "hash": "22af645d1859cb5c"
+    },
+    {
+      "timestamp": 1675379262.436703,
+      "start_timestamp": 1675379261.736703,
+      "description": "SELECT `customer`.`id` FROM `customers` WHERE `customer`.`name` = 'customerName'",
+      "op": "db",
+      "span_id": "77387b01ac144559",
+      "parent_span_id": "f32e597a998a4e49",
+      "trace_id": "9a2e3dbcabf2497b8dc79c5b7e4d0383",
+      "hash": "f47131bf6c8aef6f"
+    },
+    {
+      "timestamp": 1675379263.436703,
+      "start_timestamp": 1675379262.436703,
+      "description": "SELECT COUNT(*) FROM `customers`",
+      "op": "db",
+      "span_id": "33f759e57af94520",
+      "parent_span_id": "f32e597a998a4e49",
+      "trace_id": "9a2e3dbcabf2497b8dc79c5b7e4d0383",
+      "hash": "51dbf6dd9fe5b83f"
+    },
+    {
+      "timestamp": 1675379264.436703,
+      "start_timestamp": 1675379263.436703,
+      "description": "SELECT COUNT(*) FROM `items`",
+      "op": "db",
+      "span_id": "e179bfee853e4440",
+      "parent_span_id": "f32e597a998a4e49",
+      "trace_id": "9a2e3dbcabf2497b8dc79c5b7e4d0383",
+      "hash": "3f28575fcb3a4847"
+    }
+  ],
+  "start_timestamp": 1675379261.436703,
+  "timestamp": 1675379261.736703,
+  "title": "/consecutive-db/",
+  "transaction": "/consecutive-db/",
+  "transaction_info": {"source": "unknown"},
+  "type": "transaction",
+  "user": {
+    "email": "saloni@example.com",
+    "ip_address": "236.50.145.105",
+    "username": "saloni",
+    "name": "Saloni Dudziak",
+    "geo": {"country_code": "US", "city": "San Francisco", "region": "CA"}
+  },
```

Comment:
This is a fictitious person, right?

Reply:
Yeah, this is just to event generated when you run ./bin/load-mocks

Reply:
Which seems to be just be sentry employees